### PR TITLE
docs: Add comment for cloud sql

### PR DESCRIPTION
**Pull request by ravisiddhu**
Updating the deletion_protection flag's comment to include the usage of the new deletion_protection_enabled flag to avoid customer confusion between the 2 flags

There exists multiple pages in our documentation, which references deletion_protection (old flag), but not deletion_protection_enabled flag (our flag) in Terraform and that is confusing some customers.

## Description

Fixes #b/277877708

## Checklist

**Readiness**

- [X] Yes, **merge** this PR after it is approved

**Style**

- [X] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [X] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [X] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [X] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [X] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [X] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s):

### CODE REVIEW

File: cloud_sql/database_basic/main.tf [hunk 1]
```diff
@@ -30,5 +30,5 @@ resource "google_sql_database_instance" "instance" {
     tier = "db-f1-micro"
   }
 
-  deletion_protection = false # set to true to prevent destruction of the resource
+  deletion_protection = false # set `deletion_protection_enabled` flag to true to enable deletion protection of an instance at the GCP level. Enabling this protection will guard against accidental deletion across all surfaces (API, gcloud, Cloud Console and Terraform) by enabling the GCP Cloud SQL instance deletion protection. On the other hand, `deletion_protection` flag prevents destroy of the resource only when the deletion is attempted in terraform.
```
Comment (enocom) [inline cloud_sql/database_basic/main.tf:33]: Let's move this comment to above `deletion_protection`. Also, please wrap at ~80 characters as this shows up in the docs and horizontal scrolling is painful for our customers.

**Review by enocom — CHANGES_REQUESTED** [review]

File: cloud_sql/database_basic/main.tf (hunk 1 above)
Comment (ravisiddhu) [reply cloud_sql/database_basic/main.tf]: Done

**Review by enocom — APPROVED** [review]

**Comment by msampathkumar** [comment]
Hi @ravisiddhu @enocom Thank you for your work.

IMHO, it is redundant to update this info in a narrow scope like these sample. My suggestion: 

1. Simplify your comments(reduce it to 1 or 2 lines) 
   Example: `# set delete_protection to true, will ensure that we can only delete this instance by use of Terraform`

2. Update the API Docs and Terraform Provider(TP) documentation with detailed information. To update TF docs, please raise a PR to update https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

**Comment by ravisiddhu** [comment]
@msampathkumar I have reduced the comment size to two lines.

**Comment by msampathkumar** [comment]
Hi, you have a invisible trailing space in the comments starting with `# set `deletion_protection` to true,...` Please check & remove to pass all the tests.

**Comment by ravisiddhu** [comment]
> Hi, you have a invisible trailing space in the comments starting with `# set `deletion_protection` to true,...` Please check & remove to pass all the tests.

I have removed the trailing white space.

**Comment by msampathkumar** [comment]
/gcbrun

Re-running test!

**Pull request merged**